hca-to-scea/script_spreadsheets/hca_to_scea_test_11/E-HCAD-1.idf.txt

```diff
@@ -10,10 +10,17 @@ Person Email	m.a.haniffa@newcastle.ac.uk	mrc38@cam.ac.uk	st9@sanger.ac.uk
+Protocol Hardware								Illumina  NextSeq 500 system
+=======
 Protocol Type	sample collection protocol	enrichment protocol	enrichment protocol	enrichment protocol	enrichment protocol	nucleic acid library construction protocol	nucleic acid sequencing protocol
 Protocol Name	P-HCAD1-1	P-HCAD1-2	P-HCAD1-3	P-HCAD1-4	P-HCAD1-5	P-HCAD1-6	P-HCAD1-7
 Protocol Description	All human developmental tissue samples used for this study were obtained from the MRC– Wellcome Trust-funded Human Developmental Biology Resource.	Cell line dissociation.	Digested tissue was passed through a 100 µm cell strainer and cells were pelleted by centrifugation at 500 g for 5 min at 4°C. Cells were then resuspended in 5 ml red blood cell lysis buffer (eBioscience) and left for 5-10 min at room temperature.	Dissociated cells were stained with anti-CD45 antibody and DAPI prior to sorting. Sorting by flow cytometry was performed with BD FACSAria Fusion Flow Cytometer. CD45 positive fraction was sorted from DAPI– CD45+ gate. CD45 gating was contiguous so that no live cells were lost in sorting.	Dissociated cells were stained with anti-CD45 antibody and DAPI prior to sorting. Sorting by flow cytometry was performed with BD FACSAria Fusion Flow Cytometer. CD45 negative fraction was sorted from DAPI– CD45– gate. CD45 gating was contiguous so that no live cells were lost in sorting.	scRNA-seq libraries were prepared with the 10X Genomics Chromium single cell 3’ v2 reagent kit. Unsorted, or DAPI–CD45+ or DAPI–CD45– FACS-isolated cells were loaded onto each channel of the Chromium chip following the manufacturer’s instructions before droplet encapsulation on the Chromium controller.	Sequencing was performed on the Illumina NextSeq 500 system. The gene expression libraries were sequenced at a target depth of 50,000 reads per cell using the following parameters: Read1: 26 cycles, i7: 8 cycles, i5: 0 cycles; Read2: 91 cycles to generate 75-bp paired-end reads. BCR and TCR libraries were sequenced at a target depth of 5000 reads per cell.
 Protocol Hardware							Illumina  NextSeq 500 system
+>>>>>>> origin/master
```

Comment:
This looks like a remnant from a git merge. Please make sure the file is in order.

Reply:
I will have a look into this and remove all the merge remnants. Thanks!